bigtable_rs/src/bigtable/read_rows.rs

```diff
@@ -33,6 +35,19 @@ pub async fn decode_read_rows_response(
     Ok(rows)
 }
 
+/// Decode the stream of `ReadRowsResponse` into a stream of `Result<(RowKey, Vec<RowCell>)>>`.
```

Comment:
Trivial: Perhaps it is nice to be a bit more explicit in the comments like:
`/// Decode and flatten the stream of ....`

Reply:
done!